### DISCUSSION TITLE
Realm migration issue (One or more errors occurred. (The process cannot access the file 'C:\Users\vanle\AppData\Roaming\osu\client.db' because it is being used by another process.))

### DISCUSSION THREAD

**Discussion by pha-Z** [ppy/osu#18843] [Q&A] [answered]
Please drag the "attach_me.zip" file here
[attach_me.zip](https://github.com/ppy/osu/files/8984906/attach_me.zip)
!

i had the classic osu! client open simultaneously

**Comment by peppy** [comment]
Can you try running lazer once more? It might work a second try.

**Comment by pha-Z** [comment]
i have started to run the import from previous version right away. so i just stopped it, closed classic and lazer osu! and reopened lazer, but my skins/songs/etc were synced with where that import left off when i stopped it

**Reply by frenzibyte** [reply]
I'll just note that osu!(stable) (aka "classic") has nothing to do with lazer, so there was no need to close it.

**Reply by frenzibyte** [reply]
You can restore your database back as you had it before by removing `client.realm` in the osu!(lazer) folder, and look for the most recent `client.db` backup file in the `backups` folder and rename it back to `client.db` and move back next to where `client.realm` used to be.

**Reply by pha-Z** [reply]
rename `client.db` ... to `client.db` ? did you mean to say that i should replace the `client.realm` file with the `client.db` from the backups folder?

**Reply by frenzibyte** [reply]
There's a backed up `client.db` file in the backup folder with a timestamp attached to the filename (e.g. `client.before_final_migration_XXXXXXXXXX.db`), pick the most recent one (sort by date created) and rename it back to `client.db` and place it where `client.realm` used to be.

**Comment by pha-Z** [comment, marked as answer]
realized that lazer is on the wrong drive to begin with and there wouldnt even be enough space to import all maps. i moved everything to my other drive and let it finish importing after i freshly installed lazer again.

**Reply by peppy** [reply]
Safe to say everything is working well now?

**Reply by pha-Z** [reply]
yup
